source/jsonnet/england-wales/census_communal_establishment.jsonnet

```diff
@@ -16,6 +16,9 @@ function(region_code, census_month_year_date) {
   schema_version: '0.0.1',
   data_version: '0.0.3',
   survey_id: 'census',
+  survey: 'CENSUS',
```

Comment:
Is the value (ignoring case) always the same for `survey_id` and `survey`? If so we shouldn't be adding a new property. we should just use the existing.

Reply:
I am pretty sure we just set the `survey_id` to `census` because it is a required property nothing more. What we want is survey given that matches the claims. If anything we should remove `survey_id` or make it optional as business may still use it.

Reply:
Ok, we should keep both for now then. I think removal or making `survey_id` optional is a separate thing, as it is in the downstream payload we would need to have conversations with the dependent teams about changing that.